docker-compose.override.yml.example

```diff
@@ -0,0 +1,7 @@
+version: '2.1'
+services:
+  obs-docu:
+    build:
+      args:
+        CONTAINER_USERID: 13042
```

Comment:
Why 13042 ? Is it some random number? Why not using a more speaking constant like ``TO_BE_REPLACED``?

Reply:
yo just a random number...